src/sentry/static/sentry/app/views/settings/projectDebugFiles/debugFileRow.tsx

```diff
@@ -0,0 +1,164 @@
+import React from 'react';
+import styled from '@emotion/styled';
+
+import {t} from 'app/locale';
+import space from 'app/styles/space';
+import TimeSince from 'app/components/timeSince';
+import Tooltip from 'app/components/tooltip';
+import Tag from 'app/views/settings/components/tag';
+import FileSize from 'app/components/fileSize';
+import Button from 'app/components/button';
+import Confirm from 'app/components/confirm';
+import {IconDelete} from 'app/icons/iconDelete';
+import Access from 'app/components/acl/access';
+import {IconClock} from 'app/icons';
+
+import {getFileType, getFeatureTooltip} from './utils';
+import {DebugFile} from './types';
+
+type Props = {
+  debugFile: DebugFile;
+  downloadUrl: string;
+  onDelete: (id: string) => void;
+};
+
+const DebugFileRow = ({debugFile, downloadUrl, onDelete}: Props) => {
+  const {
+    id,
+    data,
+    debugId,
+    uuid,
+    size,
+    dateCreated,
+    objectName,
+    cpuName,
+    symbolType,
+  } = debugFile;
+  const fileType = getFileType(debugFile);
+  const {features} = data || {};
+
+  return (
+    <React.Fragment>
+      <Column>
+        <DebugId>{debugId || uuid}</DebugId>
+        <TimeAndSizeWrapper>
+          <StyledFileSize bytes={size} />
+          <TimeWrapper>
+            <IconClock size="xs" />
+            <TimeSince date={dateCreated} />
+          </TimeWrapper>
+        </TimeAndSizeWrapper>
+      </Column>
+      <Column>
+        <Name>
+          {symbolType === 'proguard' && objectName === 'proguard-mapping'
+            ? '\u2015'
+            : objectName}
+        </Name>
+        <Details>
+          {symbolType === 'proguard' && cpuName === 'any'
+            ? t('proguard mapping')
+            : `${cpuName} (${symbolType}${fileType && ` ${fileType}`})`}
+
+          {features &&
+            features.map(feature => (
+              <Tooltip key={feature} title={getFeatureTooltip(feature)}>
+                <Tag inline>{feature}</Tag>
+              </Tooltip>
+            ))}
+        </Details>
+      </Column>
+      <RightColumn>
+        <Access access={['project:releases']}>
+          {({hasAccess}) => (
+            <Button
+              size="xsmall"
+              icon="icon-download"
```

Comment:
We have a new icon component for `IconDownload`. Perhaps in a follow up pull request?

Reply:
Oof, I missed this one - thanks!